angular-apollo-tailwind/projects/shared/src/lib/pipes/generate-url-with-protocol/generate-url-with-protocol.pipe.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { GenerateUrlWithProtocolPipe } from './generate-url-with-protocol.pipe';
+
+describe('GenerateUrlWithProtocolPipe', () => {
+  it('create an instance', () => {
+    const pipe = new GenerateUrlWithProtocolPipe();
+    expect(pipe).toBeTruthy();
+  });
```

Comment:
Let's add couple of basic unit tests for that logic :)
Unit tests should be part of each new PR. Especially if we're fixing a bug